src/cli/cli-report.c

```diff
@@ -826,13 +845,16 @@ static int run_event_chain_real(struct run_event_state *run_state,
  */
 int run_event_chain(const char *dump_dir_name, GList *chain, int interactive)
 {
+    /* Expand wildcards in the chain. */
+    GList *expanded_chain = expand_event_chain_wildcards(chain);
     struct run_event_state *run_state = new_run_event_state();
-    int retval;
 
-    retval = run_event_chain_real(run_state, dump_dir_name, chain, interactive);
+    /* Run the expanded chain. */
```

Comment:
Same here, comment adds no value.

Reply:
I felt so lonely and comment-depraved that I had to make up at least these feeble grey companions of mine here and there.